<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="selectDomainLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Matrix domain:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selectDomainBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #dc143c;</string>
   </property>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="setMatrixButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt;
background-color: &quot;crimson&quot;;
color: black;
</string>
   </property>
   <property name="text">
    <string>Set matrix</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
